diff --git a/releng/airootfs/etc/motd b/releng/airootfs/etc/motd
index 4d9eda1..fb0f001 100644
--- a/releng/airootfs/etc/motd
+++ b/releng/airootfs/etc/motd
@@ -9,3 +9,5 @@ After connecting to the internet, the installation guide can be accessed
 via the convenience script [35mInstallation_guide[0m.
 
 [41m [41m [41m [40m [44m [40m [41m [46m [45m [41m [46m [43m [41m [44m [45m [40m [44m [40m [41m [44m [41m [41m [46m [42m [41m [44m [43m [41m [45m [40m [40m [44m [40m [41m [44m [42m [41m [46m [44m [41m [46m [47m [0m
+
+Start the Xfce4 desktop with the [35mstartx[0m.
