diff --git a/releng/profiledef.sh b/releng/profiledef.sh
index ce26e79..2c15c69 100644
--- a/releng/profiledef.sh
+++ b/releng/profiledef.sh
@@ -2,10 +2,10 @@
 # shellcheck disable=SC2034
 
 iso_name="archlinux"
-iso_label="ARCH_$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y%m)"
-iso_publisher="Arch Linux <https://archlinux.org>"
-iso_application="Arch Linux Live/Rescue DVD"
-iso_version="$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y.%m.%d)"
+iso_label="ARCH_$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y%m)_JP"
+iso_publisher="minetaro12 <https://0sn.net>"
+iso_application="Arch Linux Live/Rescue DVD with minetaro12 customizations"
+iso_version="$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y.%m.%d)_JP"
 install_dir="arch"
 buildmodes=('iso')
 bootmodes=('bios.syslinux.mbr' 'bios.syslinux.eltorito'
